ibm/service/iampolicy/resource_ibm_iam_policy_template.go

```diff
@@ -179,6 +177,40 @@ func ResourceIBMIAMPolicyTemplate() *schema.Resource {
 							Elem:        &schema.Schema{Type: schema.TypeString},
 							Description: "Role names of the policy definition",
 						},
+						"subject": {
```

Comment:
If its input like required /optional we shd prefer to use TypeSet instead of TypeList because list unordered items which may show diff .